full-stack-tests/core/webpack.config.js

```diff
@@ -39,10 +38,13 @@ function createConfig(shouldInstrument) {
       mainFields: ["main", "module"],
       fallback: {
         assert: require.resolve("assert"),
+        crypto: require.resolve("crypto-browserify"),
```

Comment:
where does this and the others come from? the update to the imodels clients?

Reply:
for fallback: 
- crypto: object-storage-core
- the rest arent needed anymore, will remove

externals:
 - fs: object-storage-core